### DISCUSSION TITLE
Should logo-ls still be maintainted?

### DISCUSSION THREAD

**Discussion by canta2899** [canta2899/logo-ls#2] [Ideas]
Opening this discussion in order to interact with all the logo-ls contributors and find out if something can be done in order to keep logo-ls maintained.

**Comment by neurolag** [comment]
I believe the biggest of trouble is keeping track of keeping track of the changes made in Nerd Fonts and potential requests from users on icons for files which haven't been implemented yet.

So I believe that maintaining `logo-ls` is more of a load of work than anyone wants to bother with.
Maybe some automated processes might help - which, then again, requires a lot of engineering, too.

**Comment by neurolag** [comment]
Alright, then
Shall we create an organization for maintaining `logo-ls`?

**Reply by canta2899** [reply]
I'm sorry for not responding to your previous comment, somehow I didn't get any notification.

I think we should consider creating the organization so we can start deploying to package managers (apt, Homebrew, yum, yay, etc.) and, as you mentioned earlier, find a way to keep track of changes in NF and user requests. I originally had this fork detached from the original repo by GitHub with the same aim, but I never found enough time to follow through 🤣.

I'll be very busy with other things until early 2025, so I'm on board with creating the organization and migrating everything there. However, I won't have much time to work on it until January 10th–15th. Shall we put this off for a few days and reconnect soon?

**Reply by neurolag** [reply]
No worries, that's okay 😄
Sounds fine by me!

**Reply by neurolag** [reply]
yooo
I know it's not exactly early 2025 but... are you feeling like talking about opening a GH org?

**Reply by canta2899** [reply]
Hey! Thanks for reopening this and bringing it back up. I'm really sorry for not picking this up back in early 2025. I remember being swamped with personal matters and work, kept putting it off by a week, then by a month, and here we are in mid-2026 🤣

I recently found the time to improve a bunch of things and add a couple of useful features, on top of deploying it to a few package managers. I'd say logo-ls is quite stable right now and has all the features that I personally need. One thing that is missing is broader coverage for other package managers (winget, scoop, apt, dnf, etc.), but objectively speaking, the repo has about 30 stars, and there are maybe 5 or 6 users in total who haven't complained so far.

With the current user base, managing everything under my personal account just makes things easier. I'd definitely be open to considering a GitHub organization if the tool suddenly gets more traction and a subsequent influx of issues and PRs. That said, if you’re looking to actively jump in and help out, I’d be more than happy to review and merge your contributions!

I'm also planning to push this project a little bit on some subreddits or maybe write a quick online article. Right now it's relatively unknown and 99% of the users of the original repo haven't switched to my fork (some people are still opening issues in the unmaintained repo), so I'd be happy if more people started using it. But at the same time, I reckon people have bigger fish to fry than spending days finding the perfect 'ls' tool 🤣.

However, just to wrap things up, I'd say that I am willing to open a github org with collateral repos for package managers support (such as the homebrew tap, etc.) if logo-ls reaches at least 100 stars.

**Reply by neurolag** [reply]
Thank you so much for coming back at me! :smile:
In this case, it's great to hear that this project found a new and maintained home!
I'm looking forward to where things lead to!
